packages/remix-sample/README.md

```diff
@@ -0,0 +1,13 @@
+# Remix Sample
+
+This is a sample project for Logto's Remix SDK.
+
+## Configuration
+
+You can configure the sample project by setting the following environment variables:
+
+| key              | description                        | example                         |
+| ---------------- | ---------------------------------- | ------------------------------- |
+| LOGTO_APP_ID     | The app ID of your application     | `my-app`                        |
+| LOGTO_APP_SECRET | The app secret of your application | `my-secret`                     |
+| LOGTO_ENDPOINT   | The endpoint of your Logto server  | `https://<tenant-id>.logto.app` |
```

Comment:
do we have a console tutorial available?

Reply:
LOG-111492